examples/video-chat/screens/ConnectWithRoomManagerScreen.tsx

```diff
@@ -28,9 +28,15 @@ async function getFishjamServer(
   roomName: string,
   userName: string,
 ) {
-  const url = roomManagerUrl.endsWith('/')
-    ? roomManagerUrl
-    : roomManagerUrl + '/';
+  // in case user copied url from admin panel
+  const urlWithoutParams = roomManagerUrl.replace(
+    '/*roomName*/users/*username*',
+    '',
+  );
```

Comment:
This line is added just to simplify example app usage. You can just copy URL from Cloud Panel